Location.gpx

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>

```

Comment:
was this intended to be checked in? 🤔 The commit said `update changelog`, but this isn't the changelog :D

Reply:
This was not intended to be here. I think I added by mistake. I will open another pull request to fix it. This is just for test purpose, because it has more cases in Europe then here in Brazil